app/assets/src/components/views/SampleUploadFlow/RemoteSampleFileUpload.jsx

```diff
@@ -133,6 +133,13 @@ class RemoteSampleFileUpload extends React.Component {
                 Paired files must be labeled with &quot;_R1&quot; or
                 &quot;_R2&quot; at the end of the basename.
               </li>
+              <li>
+                File names must be no longer than 120 characters and can only
+                contain letters from the English alphabet (A-Z, upper and lower
+                case), numbers (0-9), periods (.), hyphens (-) and underscores
+                (_). Spaces are not allowed.
+              </li>
+              <li>File names must be no longer than 140 characters.</li>
```

Comment:
This line is not included above? Any particular reason?

Reply:
No, actually that line should have been removed, since the previous item already includes it. Thank you for pointing it out.